test/integration/scheduler_perf/util.go

```diff
@@ -33,8 +35,28 @@ import (
 	_ "k8s.io/kubernetes/plugin/pkg/scheduler/algorithmprovider"
 	"k8s.io/kubernetes/plugin/pkg/scheduler/factory"
 	"k8s.io/kubernetes/test/integration/framework"
+	"strconv"
 )
 
+var running_metrics_server bool
+
+// setupMetricsServer starts a metrics server so we can monitor scheduling latency/throughput/etc
+func setupMetricsServer() {
+	if !running_server {
+		running_server = true
+		glog.V(2).Info("starting metrics server")
+		mux := http.NewServeMux()
+		mux.Handle("/metrics", prometheus.Handler())
+		server := &http.Server{
+			Addr:    net.JoinHostPort("127.0.0.1", strconv.Itoa(int(9091))),
+			Handler: mux,
```

Comment:
Nits: 
Should we make it configurable? Will this work if 9091 port is being used by some other process not related to Kube?
Do we need to convert from int into string, Can we use "9091") - if we don't want that to be configurable?